parlai/mturk/tasks/acute_eval/analysis.py

```diff
@@ -412,19 +472,47 @@ def _render_row(matchup: List[str], row: pd.Series) -> str:
                     + ''.join(result)
                     + '</div>'
                 )
-            return f"<tr><td>{dialogues['winner_dialogue']}</td><td>{dialogues['loser_dialogue']}</td><td>{row['reason']}</td></tr>"
+
+            checkbox = (
+                '<div><input type= "checkbox" id= "cherry" name= "cherry">'
+                '<label for="cherry">Cherry</label>'
+                '</div>'
+                '<div><input type= "checkbox" id= "lemon" name= "lemon">'
+                '<label for= "lemon">Lemon</label>'
+                '</div>'
+                '<div><input type= "checkbox" id= "neutral" name= "neutral">'
+                '<label for= "neutral">Neutral</label>'
+                '</div>'
+            )
+            if row['winner'] == row['speaker_model_mapping'][0]:
+                speakers_footnote = "(Speaker_1 = {}(winner), Speaker_2 = {})".format(
+                    row['speaker_model_mapping'][0], row['speaker_model_mapping'][1]
+                )
+            else:
+                speakers_footnote = "(Speaker_1 = {}, Speaker_2 = {}(winner))".format(
+                    row['speaker_model_mapping'][0], row['speaker_model_mapping'][1]
+                )
+            return f"<tr><td>{'Pair ' + str(row_id)}</td><td>{checkbox}</td><td>{dialogues['winner_dialogue']}</td><td>{dialogues['loser_dialogue']}</td><td>{row['reason']+chr(10)+speakers_footnote}</td></tr>"
```

Comment:
would you mind breaking into a multiline string? Also a little confused about the string concatenation inside the f-string.

Reply:
Good catch! I'll fix that.